docs/services/oceanoptics_spectrometer.rst

```diff
@@ -0,0 +1,42 @@
+Ocean Optics Spectrometer
+====================
+
+This service controls an Ocean Optics Spectrometer. It is a wrapper around the python-seabreeze package.
+The python-seabreeze package needs to be installed first, and the page also explain how to install the
```

Comment:
Very happy you pointed this out because I forgot ^^: We need to add `conda-forge::seabreeze` to the `environment.yml`! And then you can rewrite the sentence here to something like: "Spectrometer drivers need to be installed for this service, which is explained in the python-seabreeze documentation."